DAQ_FW/lib/IADCSensor/IADCSensor.h

```diff
@@ -0,0 +1,31 @@
+#ifndef IADCSENSOR_LIB
+#define IADCSENSOR_LIB
+
+#include "Logger.h"
+#include "system_config.h"
+
+class IADCSensor
```

Comment:
for general cleanliness, class definitions should be organized in terms of relevance to an outside user. If I'm trying to use this class, I really only care about the constructor and read functions, not the stuff I can't even access like private members. So here's a suggested order. It kinda varies depending on who's code style you follow but at the very least constructors/destructors and then public methods should be first 

```
public: 
     constructors/destructors
     methods
     members
    
private:
    members 
    methods

```